packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/lambda-walkthrough.ts

```diff
@@ -64,45 +66,47 @@ export async function createWalkthrough(
   // ask template selection questions and merge in results
   templateParameters = merge(templateParameters, await templateWalkthrough(context, templateParameters));
 
-  // list out the advanced settings before asking whether to configure them
-  context.print.info('');
-  context.print.success('Available advanced settings:');
-  advancedSettingsList.forEach(setting => context.print.info('- '.concat(setting)));
-  context.print.info('');
+  if(!templateParameters.skipAdvancedSection) {
```

Comment:
nit: can we reverse this condition and early return to avoid more nesting?

Reply:
Updated as suggested. Also, updated the skipNextSteps method to follow the same code style.